packages/demo/scss/index.scss

```diff
@@ -61,6 +61,26 @@
     white-space: normal;
 }
 
+.docs-Placeholder {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    padding: 48px 24px;
```

Comment:
Rest of padding in this file is in 'rem', shouldn't this be too?

Reply:
updated